### ISSUE TITLE
question resumed on #30

### ISSUE THREAD

**Issue by dadada101** [openseg-group/openseg.pytorch#70]
@hsfzxjy ,really thanks for your prompt and detailed reply!!!
I got your meaning . Because mIou is a metric insensitive to the class imbalance problem in the ground truth label, right? so at the begining, I thought miou is a better metric than accuray...
in addiction thanks for your training trick advise, may I ask why adopting a smaller crop than segmentation model is benificial?
---------------
#30 https://github.com/openseg-group/openseg.pytorch/issues/30
